test/src/e2e_vm_tests/test_programs/should_fail/associated_fn_params_as_method_call/Forc.toml

```
@@ -0,0 +1,8 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+license = "Apache-2.0"
+name = "associated_fn_params_as_method_call"
+entry = "main.sw"
+
+[dependencies]
+core = { path = "../../../../../../sway-lib-core" }
```

Same thing here, for faster test compilation we can omit the dependenecy.
```suggestion
entry = "main.sw"
implicit-std = false
```